sass/_DeskHead.scss

```diff
@@ -0,0 +1,259 @@
+@media only screen and (min-width: 768px) {
+  .mobile,
+  .btn__style,
+  .foto__email,
+  .foto__email__links,
+  .incdiactor__img,
+  .bgrht__img,
+  .bgimg,
+  .mobile-form {
+    display: none;
+  }
+
+  .footer__container {
+    background-image: none;
+  }
+
+  .hug {
+    display: none;
+
+    &__img {
+      display: none;
+    }
+  }
+
+  .mobile__label,
+  .mobile__input,
+  .mobile__textarea {
+    display: none;
+  }
+
+  .desktop {
+    margin-top: 2rem;
+  }
+
+  .desktop__nav {
+    display: flex;
+    justify-content: center;
+
+    &__list {
+      list-style: none;
+    }
+
+    &__link {
+      text-decoration: none;
+      padding: 20px;
+      color: #28352f;
+      font-family: inherit;
+      font-size: 20px;
+      font-weight: 600;
+      line-height: 28px;
+      letter-spacing: 0;
+      text-align: center;
+    }
+  }
+
+  .headline__secondary {
+    margin: auto;
+    width: 55%;
+    text-align: center;
+  }
+
+  .Main-container {
+    background-image: url(../img/Header-llustration-desktop@2x.png);
+    background-repeat: no-repeat;
+    background-size: 100% 50%, contain;
+    background-position: 0 -20px;
+  }
+
+  .hug__Desktop {
+    text-align: center;
+    margin-top: 20px;
+
+    &__img {
+      height: 50px;
+      width: 50px;
+      left: 144px;
+      top: 4px;
+      border-radius: 32px;
+      padding: 8px;
+    }
+  }
+
+  .recent__work {
+    margin: auto;
+    width: 55%;
+    text-align: center;
+  }
+
+  .works {
```

Comment:
Kindly note that both the mobile and desktop versions need to use grid to arrange this cards.
It is a requirement for you to pass this section
<img width="1680" alt="Screenshot 2022-09-30 at 04 41 47" src="https://user-images.githubusercontent.com/48546992/193257553-a85611bc-7fbc-44b6-ba23-b7c9aea0798a.png">

Reply:
Hello @Nyame-Wolf The text-align:center is not a design for the cards; there were design using grid; this is for the heading "Recent Work"